app/models/components/ability_host.rb

```diff
@@ -83,6 +83,17 @@ def currently_valid_hashes
         }
       ]
     end
+
+    # Returns a condition which will return started rows(start_at before current time) when
+    # ORed together in a database query. Reverse-merge this with your conditions to obtain the
+    # set of already started rows in the table.
+    #
+    # @return [Hash] The hash condition.
+    def already_started_hash
```

Comment:
In v1 the expired ones do not show in course home page(the page with activities).

And in v2 it mostly just a indicator now ( don't have course home page yet)?  I mean the we have the calendar icon and a different css styles for expired ones.